packages/mobile-app-did/js/pages/CryptoGift/GiftDetail/index.tsx

```diff
@@ -0,0 +1,200 @@
+import PageContainer from 'components/PageContainer';
+import Svg from 'components/Svg';
+import Touchable from 'components/Touchable';
+import { useLanguage } from 'i18n/hooks';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { FlatList } from 'react-native-gesture-handler';
```

Comment:
why import `FlatList` from `react-native-gesture-handler`? shouldn't it be `react-native`?

Reply:
in this situation, both are ok. [react-native-gesture-handler](https://github.com/software-mansion/react-native-gesture-handler) is more performant with gestures in comparison to react-native's, as they run on the native thread instead of the JS thread.

Reply:
> in this situation, both are ok. [react-native-gesture-handler](https://github.com/software-mansion/react-native-gesture-handler) is more performant with gestures in comparison to react-native's, as they run on the native thread instead of the JS thread.

Got it.